commit cfe9f1cf2597850233827640b6e654be5ee2742a (refs/changes/21/621/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-07T09:39:08-07:00 (3 years ago)
    
    DOCKER-945 docker service not starting due to uncaught type error

diff --git a/package.json b/package.json
index 51fd7cc..014dd2b 100644
--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
     "fwrule": "git+ssh://git@github.com:joyent/sdc-fwrule.git#d1174be",
     "imgmanifest": "git+ssh://git@github.com:joyent/node-imgmanifest.git#91e5d80",
-    "keyapi": "git+ssh://git@github.com:joyent/keyapi.git#0a49656",
+    "keyapi": "git+ssh://git@github.com:joyent/keyapi.git#c30dd2710ad2175095dc0e96479686fa774b8063",
     "libuuid": "0.1.3",
     "lru-cache": "2.5.0",
     "lstream": "0.0.4",
